pom.xml

```diff
@@ -4,7 +4,7 @@
     <modelVersion>4.0.0</modelVersion>
     <groupId>nrjmx</groupId>
     <artifactId>nrjmx</artifactId>
-    <version>1.4.2</version>
+    <version>0.0.0</version>
```

Comment:
is it ok the version ☝️

Reply:
we inject the version on build, so no need to update this line everytime